Demos/Blazorise.Demo/Layouts/MainLayout.razor

```diff
@@ -109,16 +109,128 @@
                                 </BarDropdownMenu>
                             </BarDropdown>
                         </BarItem>
+                        <BarItem>
+                            <BarDropdown>
+                                <BarDropdownToggle>
+                                    <BarIcon IconName="IconName.CompactDisc" />
+                                    Datagrid
+                                </BarDropdownToggle>
+                                <BarDropdownMenu>
+                                <BarDropdown>
+                                    <BarDropdownToggle>
+                                        Data
+                                    </BarDropdownToggle>
+                                    <BarDropdownMenu>
+                                        <BarDropdownItem To="tests/datagrid/static">
+                                            In Memory
+                                        </BarDropdownItem>
+                                        <BarDropdownItem To="tests/datagrid/manual">
+                                            From an External Source
+                                        </BarDropdownItem>
+                                    </BarDropdownMenu>
+                                </BarDropdown>
+                                <BarDropdown>
```

Comment:
I don't think we need every category to be a submenu. It can just be an item with a direct link.

Reply:
Planning on having feature examples inside each section. If we dont end up doing feature examples inside some of the sections, we change it to direct links, yes.

Reply:
Data and Selection are currently the only sections that have feature examples inside of them.

Reply:
Ah OK. There will be a lot of examples I can see :)